Drivers/Src/LTC6811.c

```diff
@@ -72,12 +72,34 @@ Copyright 2017 Linear Technology Corp. (LTC)
 #include "config.h"
 #include "BSP_SPI.h"
 #include "os.h"
+#include "BSP_OS.h"
 #include "Tasks.h"
 
 /*********************************************************/
 /*** Code that was added by UTSVT. ***/
 /*********************************************************/
 OS_MUTEX MinionsASIC_Mutex;
+bsp_os_t spi_os;
+OS_SEM MinionsIO_Sem4;
+
+#define RTOS 
+// RTOS Setup
+void LTC6811_Pend(void) {
+    CPU_TS ts;
+    OS_ERR err;
+    OSSemPend(&MinionsIO_Sem4,
+                        0,
+                        OS_OPT_PEND_BLOCKING,
+                        &ts,
+                        &err);
+}
```

Comment:
Inside these functions, you should also call the assertOS function(not sure if that is the correct name but it should be in tasks.c.